### DISCUSSION TITLE
Dynamic Task Mapping: Generates twice as many tasks as items in array

### DISCUSSION THREAD

**Discussion by tagoad** [apache/airflow#23761] [Q&A]
I didn't want to create an issue just incase its something on my end, but has anyone seen this happening? It seems to only happen when i go from one dynamic mapped task to another, the second task generates twice the amount of tasks it should and then the extras error out because the index is out of range for the source data.

**Comment by josh-fell** [comment]
Hi @tagoad! Would you be able to share a code snippet that demonstrates what you're seeing? It would be super helpful to grasp the scenario and assist.
